docker/s6/.s6-svscan/finish

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec /bin/true
```

Comment:
## Init 3 Script

This script is run when the docker container is stopped (shutdown process), this is required by s6 an has no purpose in our case that's why it only calls `/bin/true`